src/__tests__/index.test.js

```diff
@@ -0,0 +1,16 @@
+import { Provider } from 'react-redux';
+import { BrowserRouter as Router } from 'react-router-dom';
+import { render, act } from '@testing-library/react';
+import App from '../App';
+import store from '../redux/configureStore';
+
+test('Should match the snapshot', async () => {
+  const dom = render(
+    <Provider store={store}>
```

Comment:
The project uses Redux for state management and data retrieved from the API are stored in the Redux store correctly 😉 Fantastic job!